fe/fe-core/src/main/java/com/starrocks/http/HttpResultSender.java

```diff
@@ -119,15 +119,23 @@ public void sendShowResult(ShowResultSet resultSet) throws IOException {
     // BE already transferred results into json format, FE just need to Forward json objects to the client
     private void writeResultBatch(TResultBatch resultBatch, ChannelHandlerContext channel, Coordinator coord) {
         int rowsSize = resultBatch.getRowsSize();
+        int count = 0;
         for (ByteBuffer row : resultBatch.getRows()) {
             // when channel is not writeable, sleep a while to balance read/write speed to avoid oom
             while (!channel.channel().isWritable()) {
                 // if channel is closed, cancel query
                 if (!channel.channel().isActive()) {
                     coord.cancel("channel is closed, cancel query");
+                    LOG.warn("http query:channel is closed, cancel query");
```

Comment:
is it possible to log more info about the query, e.g. the query ID in the message.

Reply:
> is it possible to log more info about the query, e.g. the query ID in the message.

fixed